activesupport/lib/active_support/core_ext/enumerable.rb

```diff
@@ -17,7 +17,7 @@ module Enumerable
   # The default sum of an empty list is zero. You can override this default:
   #
   #  [].sum(Payment.new(0)) { |i| i.amount } # => Payment.new(0)
-  def sum(identity = nil, &block)
+  def _enum_sum(identity = nil, &block)
```

Comment:
I think it is not good to define public method even if method name starting with `_`.
And I feel inlining this method is better than a private method.